### ISSUE TITLE
how to set explicitArray=false only for non-child node

### ISSUE THREAD

**Issue by shoysal** [Leonidas-from-XIV/node-xml2js#646]
I have a object class
payment {
	details [
		{
			FirstName:"goo",
			LastName:"foo"
		},
		{
			FirstName:"poo",
			LastName:"shoo"
		}
	]
}
I want details object to always be an array even if I have just 1 object in the list.
If I set explicitArray=false then if the detail has one object it converts array into an object. I am getting a error in the code as its expecting an array of object and not an object.
Is there a way to set explicitArray=false only for non-child node.

**Comment by Leonidas-from-XIV** [comment]
No, you need to set `explicitArray` to true and (possibly) post-process.
